shumway/__init__.py

```diff
@@ -87,14 +92,13 @@ def __exit__(self, *args):
 
 class MetricRelay(object):
     """Create and send metrics"""
-    def __init__(self, default_key, ffwd_ip=FFWD_IP, ffwd_port=FFWD_PORT,
-                 default_attributes=None):
+    def __init__(self, default_key, ffwd_host=FFWD_IP, ffwd_port=FFWD_PORT,
+                 ffwd_path=None, default_attributes=None, use_http=False):
```

Comment:
The `use_http` flag should likely have it's own use case in the readme.

Reply:
I added a section about using the lib with http